src/Vegetation/Roots.jl

```diff
@@ -160,9 +161,11 @@ and soil pressure at the root tips, or with a dynamic soil model using `ClimaLSM
 
 $(DocStringExtensions.FIELDS)
 """
-struct RootsModel{FT, PS, D, RE, T} <: AbstractVegetationModel{FT}
+struct RootsModel{FT, PS, PSE, D, RE, T} <: AbstractVegetationModel{FT}
     "Parameters required by the root model"
     param_set::PS
+    "Physical Constants and other clima-wide parameters"
+    earth_param_set::PSE
```

Comment:
I think the intent of CLIMAParameters is that each model will only need a single param set stored. This will be constructed via a mixture of both "clima-wide" parameters and model specific ones. See [these examples](https://clima.github.io/CLIMAParameters.jl/previews/PR63/parameter_structs/).

Reply:
Perhaps what you have here is a fine start point, and we can define `RootParameters`, etc, later this month.

Reply:
stuck everything in one `parameters` object as discussed. I did not organize as parameters.soil_params and parameters.earth_params, but rather like:
```
struct Parameters{FT, PS}
a::FT
b::FT
earth_param_set::PS
end
```

Reply:
it looks *way* cleaner! it's very freeing to not think about the EnergyHydrology model for soil as composed of a "heat model" and a "soil model", (with "soil heat params" and "soil water params" but just as its own thing ("energyhydrology params").